### ISSUE TITLE
Film reported as "missing" even though download was successful. 

### ISSUE THREAD

**Issue by cflannagan** [Radarr/Radarr#2793]
When I checked library this morning, I noticed a new film was added (it was previously in my "wanted" list.  (it was the first successful automatic download for the Radarr I'm using for first time). 

However, in "movie info" in Radarr, the film is still reported as missing. 

I checked download client and it appears the film was downloaded twice - and as far as I can tell, the first download was actually successful too, so I presume it did the 2nd download because it assumed the movie was missing. 

I used "scan disk" but it still did not pick up that the film is there. 

Radarr does have read/write on the movies folder. "Bulk import movies" worked flawlessly for me on that same folder, as well as renaming and moving the film into that folder (does actions correctly). 

To prevent Radarr from attempting to add film yet again, I changed the setting to "not monitored"

**Describe the bug**
Radarr reports movie as missing after successfully adding new movie into library.

**To Reproduce**
1. Add a movie into "wanted" list
2. Wait until Radarr adds that movie
3. Go to movie info in Radarr for that movie

**Expected behavior**
There should not be "missing" tag if the movie was added successfully

**Comment by galli-leo** [comment]
You also need to provide debug logs.

**Comment by cflannagan** [comment]
@galli-leo 

So from your perspective this is not a bug? If it indeed is not a bug, I'll take it to support. 

Otherwise, I'll provide debug logs. Let me know and thanks.

**Comment by TalZiv** [comment]
Check if your movie files have the year in them.

**Comment by cflannagan** [comment]
If you're referring to the movie folder, it does have year (it's in this format: `Movie Name (2018)` (in settings: `{Movie TitleThe} ({Release Year})`

The actual movie files inside the folder also has year in file name - in this format: `{Movie.Title}.{Release Year}.{Quality.Full}.{MediaInfo.Audio}.{MediaInfo.video}-{Release Group}`

What's odd is that I never have had this issue when I have movies added into library manually via Radarr using `Search for Movie` functionality, but for movies added automatically by Radarr due to being on my "wanted" list, I ran into this issue. Granted, this is the first and only movie to have been added in this way, so it might have been a fluke. I'll be keeping eye on future movies added automatically and also provide debug logs next time if it happens again.
